src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps2_mqtt_client_module.c

```diff
@@ -100,24 +100,38 @@ static ADPS_MQTT_CLIENT_MODULE_STATE* ModuleStateFromModuleHandle(ADUC_AGENT_MOD
 ADUC_AGENT_MODULE_HANDLE ADPS_MQTT_Client_Module_Create()
 {
     ADUC_AGENT_MODULE_HANDLE handle = NULL;
+
     ADPS_MQTT_CLIENT_MODULE_STATE* moduleState = NULL;
```

Comment:
What does "ADPS_" stand for?

Reply:
@Nox-MSFT Would you know what the 'A' in ADPS stands for? I'm guessing it's short for either Azure or ADU